packages/itwin/geo-tools/src/components/GeoAddressSearch.scss

```diff
@@ -5,22 +5,26 @@
 
 
 .geotools-geoaddresssearch__container {
-  width: 300px
+  width: 300px;
+
+  ._iui3-end-icon {
```

Comment:
We should avoid overriding styles in iTwinUi. Change on their side could unknowingly break us

Reply:
I have removed the iTwinUI change now. Will request a change on their end to allow hiding the end icon.